### PR TITLE
Add dev-story with all triggers of tooltip

### PR DESCRIPTION
**Pull request by jamesricky**
Using clickable and non-clickable elements.

### CODE REVIEW

File: storybook/src/admin/Tooltip.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    },
+
+    name: "Icon with Tooltip",
+};
+
+export const AllTooltipTriggers = {
+    render: function Render() {
+        const [clicks, setClicks] = useState(0);
+
+        const NonClickable = forwardRef<HTMLDivElement, { children: React.ReactNode }>(({ children }, ref) => {
```
Comment (johnnyomair) [inline storybook/src/admin/Tooltip.stories.tsx:26]: Please add an import for ReactNode.

Comment (johnnyomair) [inline storybook/src/admin/Tooltip.stories.tsx:26]: Why do you need the ref forwarding here?

File: storybook/src/admin/Tooltip.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                <Add />
+            </Tooltip>
+        );
+    },
+
+    name: "Icon with Tooltip",
+};
+
+export const AllTooltipTriggers = {
+    render: function Render() {
```
Comment (johnnyomair) [inline storybook/src/admin/Tooltip.stories.tsx:23]: I'd rather ignore the SonarCloud error for now and configure it correctly in the future.

File: storybook/src/admin/Tooltip.stories.tsx (hunk 1 above)
Comment (jamesricky) [reply storybook/src/admin/Tooltip.stories.tsx]: Ah sorry, tried something out and forgot to remove that 😅

File: storybook/src/admin/Tooltip.stories.tsx (hunk 2 above)
Comment (jamesricky) [reply storybook/src/admin/Tooltip.stories.tsx:23]: I agree but this prevents the PR from being merged, that's why I added this

Comment (johnnyomair) [reply storybook/src/admin/Tooltip.stories.tsx:23]: This will be fixed by https://github.com/vivid-planet/comet/pull/2937, so I'd revert it.

Comment (jamesricky) [reply storybook/src/admin/Tooltip.stories.tsx]: Done in 9f5aacc097c02534fd006688477c8bf79f10721a

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**